tasks/13_largest-of-four/largest-of-four.js

```diff
@@ -0,0 +1,11 @@
+export const largestOfFour = (arr) => {
+  return arr.map( (cur) => {
+    let maxValue = cur[0];
+    for (let i = 0; i < arr.length; i++) {
```

Comment:
In case of using cur you have a mistake `i < arr.length;` should be `i < cur.length;` but it is better to use `Math.max` with `...cur` or `Math.max.apply(null, cur)`